src/tracked-models/adapt-tracked-data.ts

```diff
@@ -0,0 +1,111 @@
+/*!
```

Comment:
This was restored and not originally created in this PR. Originally deleted as one of the big cleanup tickets where duplicated ids problems were also tackled.